readme.md

```diff
@@ -64,7 +64,7 @@ Click the type names for complete docs.
 
 ### Utilities
 
-- [`Omit`](source/omit.d.ts) - Create a type from an object type without certain keys.
+- [`Except`](source/except.d.ts) - Create a type from an object type without certain keys.
```

Comment:
I think you should mention that this is a stricter version than the built-in `Omit`.